docs/openapi/lpa-codes-openapi-v1.yml

```diff
@@ -40,7 +40,7 @@ paths:
         uri: arn:aws:apigateway:${region}:lambda:path/2015-03-31/functions/arn:aws:lambda:${region}:${account_id}:function:$${stageVariables.lpa_codes_function_name}/invocations
         responses:
           default:
-            statusCode: "200"
+            statusCode: "201"
```

Comment:
Not that it really matters, but since this doesn't create a resource you can access a `200` probably makes more sense.

Reply:
I went with 201 because the endpoint is called 'create' and you are generating a new entity in the DB.